util/diff_runs.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python3
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tool for pulling 2 results JSON blobs for given runs, and diffing the results.
+
+Example usage:
+./diff_runs.py \
+    --before=chrome@b952881825 \
+    --after=chrome@7a0cf8ade7
+"""
+
+import argparse
+import inspect
+import json
+import logging
+import math
+import os
+import urllib3
+
+from urllib.parse import urlencode
+from typing import List
+
+
+here = os.path.dirname(__file__)
+
+
+def main():
+    args = parse_flags()  # type: argparse.Namespace
+
+    loggingLevel = getattr(logging, args.log.upper(), None)
+    logging.basicConfig(level=loggingLevel)
+
+    beforeSHA = args.before.sha  # type: str
+    afterSHA = args.after.sha  # type: str
```

Comment:
Are these picked up by a tool?

Reply:
Yah - it's PEP 484, which is picked up in (some?) python tools + IDEs.